Resources/Prototypes/_NF/Entities/Clothing/Belt/belts.yml

```diff
@@ -0,0 +1,12 @@
+- type: entity
+  parent: ClothingBeltStorageBase
+  id: ClothingBeltArcadia
+  name: arcadia webbing
+  description: A webbing created by Arcadia Industries. Seems very capable of fitting many items.
+  components:
+- type: Sprite
+  sprite: _NF/Clothing/Belt/arcadia-webbing.rsi
+- type: Clothing
+  sprite: _NF/Clothing/Belt/arcadia-webbing.rsi
+- type: Storage
+  capacity: 60
```

Comment:
yml is sensitive to indentation, so everything under the `  components:` needs to get formatted with 1 more double space indent